lib/cookbook-release/release.rb

```diff
@@ -0,0 +1,107 @@
+class Release
+
+  attr_reader :git
+
+  def initialize(git)
+    @git = git
```

Comment:
Maybe you could provide a default value for this `git` parameter most of the time `GitUtilities.new` seems to be enough